### ISSUE TITLE
HttpRequestException with 403 Forbidden on startup

### ISSUE THREAD

**Issue by alexb5dh** [Wox-launcher/Wox#620]
Wox queries **https://api.github.com/repos/Wox-launcher/Wox/releases** on startup which has pretty tight limits on requests per time period. If launched too often this leads to continuos **HttpRequestException** as it keeps pinging untill success.

Log:

```
System.Net.Http.HttpRequestException: Response status code does not indicate success: 403 (Forbidden).
   Source: System.Net.Http
   TargetAssembly: System.Net.Http, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
   TargetModule: System.Net.Http.dll
   TargetSite: System.Net.Http.HttpResponseMessage EnsureSuccessStatusCode()
   at System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode()
   at Squirrel.UpdateManager.<GitHubUpdateManager>d__2b.MoveNext() in c:\Users\paulb\code\squirrel\squirrel.windows\src\Squirrel\UpdateManager.Factory.cs:line 72
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Wox.App.<OnActivated>d__13.MoveNext() in C:\projects\wox\Wox\App.xaml.cs:line 79
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

**Wox**: 1.3
**Windows 8.1**

**Comment by bao-qian** [comment]
the api can be accessed 60 times per hour, so just wait would solve the problem...

**Comment by alexb5dh** [comment]
I noticed that it checks for update not only on startup but on every activation so this limits can be reached pretty quick on regular use case.
According to stacktrace it looks more like a Squirrel issue so it probably should be reported on their repo but as a temporarily fix update check can be wrapped in try/catch or similar.

**Comment by bao-qian** [comment]
correct, I will fix this once I fixed #614
